purchase_create_invoice_restrict/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Users with the group_account_invoice or group_create_bill_allowed can press the "CREATE BILL" button.
```

Comment:
```suggestion
This module adds a new group group_create_bill_allowed, and hids "Create Bill" button of the purchase order form for users without group_account_invoice or group_create_bill_allowed.

This functionality intends to prevent the situation where purchase users accidentally create vendor bills from purchase orders, when bills are supposed to be created by authorized users.
```